src/main/java/bio/terra/cli/service/utils/DataRepoService.java

```diff
@@ -81,12 +82,12 @@ public RepositoryConfigurationModel getVersion() {
    */
   public RepositoryStatusModel getStatus() {
     UnauthenticatedApi unauthenticatedApi = new UnauthenticatedApi(apiClient);
-    RepositoryStatusModel status = null;
     try {
-      status = unauthenticatedApi.serviceStatus();
+      return unauthenticatedApi.serviceStatus();
     } catch (Exception ex) {
+      // catch any exception, including client-side ones (e.g. UnknownHostException)
       logger.error("Error getting Data Repo status", ex);
+      return null;
```

Comment:
consider adding a comment about why it's helpful to swallow the getStatus exception: to allow returning info from the other exceptions. Or, throw the exception and handle the swallowing higher up for the status command specific case. ditto other status endpoints. Or, encode it in the return value of each status endpoint, since we want to treat api errors as equivalent to bad statuses.

Reply:
Moved the exception swallowing up one level into ServerManager, so that the *Service classes don't have any CLI-logic in them. Also added a comment about the reason for this logic.